### DISCUSSION TITLE
[RFC] New Design of Calendar

### DISCUSSION THREAD

**Discussion by zombieJ** [ant-design/ant-design-mobile#6009] [RFC]
![1652255909955-19e8ad16-2d67-4dee-afe3-35afeb410988](https://user-images.githubusercontent.com/5378891/223048713-a38154cd-b940-4f20-a62a-d2ca1e8063d2.jpg)

**Comment by asevenm** [comment]
新的设计大概什么时候可以上线呢

**Comment by Me-UserName** [comment]
新的设计大纲什么时候可以上线呢?

**Comment by miracles1919** [comment]
#6173 快了
